CondFormats/DataRecord/interface/HGCalMappingModuleRcd.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CondFormatsModuleRcd_HGCalMappingModuleRcd_h
+#define CondFormatsModuleRcd_HGCalMappingModuleRcd_h
+// -*- C++ -*-
+//
+// Package:     CondFormats/DataRecords
+// Class  :     HGCalMappingModuleRcd
+//
+/**\class HGCalMappingModuleRcd HGCalMappingModuleRcd.h CondFormats/DataRecords/interface/HGCalMappingModuleRcd.h
```

Comment:
There is an extra *

Reply:
@jfernan2 thanks - for this and the previous just a comment - they have been generated automatically with `mkrecord`. We can remove the *, but maybe `mkrecord` should be patched?

Reply:
Why would the second `*` be a problem? The `mk*` scripts have produced this kind of comments since ~ever (and I also personally follow similar convention for multi-line documentation for classes and functions), and I'd prefer to not change the `mk*` templates without a good reason.

Reply:
I think at some point (recently) gitHub changed the web-based rendering of these commenting convention such that they don't look like comment anymore (it's a bit bothering but not sure it warrants a change in the mk* template).

Reply:
With some testing https://github.com/makortel/test/blob/main/comment.cc it seems it is the
```c++
//
/*
```
that "breaks" the rendering. The second `*` in `/**` makes no difference. A newline between the two different comment styles "fixes" the rendering
```c++
//

/*
```
In any case I think this question should be addressed (if addressed) separately from this PR (and more generally).